content/en/platform/corda/5.0/key-concepts/cluster-admin/workers/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Workers"
+date: 2023-07-24
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-cluster-admin-workers
+    parent: corda5-key-concepts-cluster-admin
+    weight: 2000
+section_menu: corda5
+---
+
+# Workers 
+
+Corda 5 introduces a distributed architecture for the Corda runtime that is based on worker processes. There are different types of worker processors which each have their own operational responsibility and because each worker is stateless, it is possible to scale horizontally.
```

Comment:
```suggestion
Corda 5 introduces a distributed architecture for the Corda runtime that is based on worker processes. There are different types of worker processors which each have their own operational responsibility and, because each worker is stateless, it is possible to scale horizontally.
```